### PR TITLE
Adding support for WP_Comment_Query

### PR DESCRIPTION
**Pull request by JasonHoffmann**
Unit tests have been organized into their own folder.

### CODE REVIEW

File: includes/batches/class-batch-comments.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+	/**
+	 * Default args for the query.
+	 *
+	 * @var array
+	 */
+	public $default_args = array(
+		'number' => 10,
+		'offset' => 0,
+		'no_found_rows' => false,
```
Comment (jjeaton) [inline includes/batches/class-batch-comments.php]: Is `no_found_rows` responsible for determining the total count? If so, would we need to keep this? (and maybe on other queries too?)

Comment (jjeaton) [inline includes/batches/class-batch-comments.php]: Ah, so by setting it to false, that's what we're doing right?

Comment (JasonHoffmann) [inline includes/batches/class-batch-comments.php]: If `no_found_rows` is set to true, which is the default, then `$query->found_comments` will always return `0`. The only way I can think of to get the total number of results (similar to `found_posts`) in comments is to set that to false, even though it slows down the query a bit.

**Comment by JasonHoffmann** [comment]
Fixes #42

**Pull request merged**